backend/src/auth/auth.module.ts

```diff
@@ -0,0 +1,28 @@
+import { Module } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { PassportModule } from '@nestjs/passport';
+import { AuthController } from './auth.controller';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { JwtModule } from '@nestjs/jwt';
+import { JwtStrategy } from './jwt.strategy';
+import { OAuth2GoogleStrategy } from './oauth2-google-strategy.service';
+import { UsersModule } from 'src/users/users.module';
+
+@Module({
+  imports: [
+    PassportModule.register({ defaultStrategy: 'jwt' }),
+    ConfigModule,
+    JwtModule.registerAsync({
+      imports: [ConfigModule],
+      useFactory: async (configService: ConfigService) => ({
+        secret: configService.get<string>('JWT_SIGNING_KEY'),
+        signOptions: { expiresIn: '48h' },
```

Comment:
**issue:** we don't want to make users sign back in every two days so let's remove the option to not add expiration to the tokens